alignak/satellite.py

```diff
@@ -83,6 +83,7 @@
 from alignak.daemon import Daemon
 from alignak.stats import statsmgr
 from alignak.check import Check  # pylint: disable=W0611
+from alignak.objects.module import Module  # pylint: disable=W0611
```

Comment:
This import is already imported here : https://github.com/Alignak-monitoring/alignak/blob/develop/alignak/bin/__init__.py. I don't think this was necessary.